web/src/main/java/com/synopsys/integration/alert/web/api/job/JobIdsValidationModel.java

```diff
@@ -0,0 +1,23 @@
+package com.synopsys.integration.alert.web.api.job;
+
+import java.util.List;
+import java.util.UUID;
+
+import com.synopsys.integration.alert.common.rest.model.AlertSerializableModel;
+
+public class JobIdsValidationModel extends AlertSerializableModel {
```

Comment:
By the name of this I thought this was the validation result of the jobs.  Can we make a slight change to make it clear this is to be used with a request.
For example: JobsIdsValidationRequestModel